<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SwissHCC_ConfigureForm</class>
 <widget class="QWidget" name="SwissHCC_ConfigureForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Template:</string>
       </property>
       <property name="buddy">
        <cstring>templateComboBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="templateComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="loadChipsCheckBox">
     <property name="text">
      <string>Load Chips</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="runProbeFindingCheckBox">
     <property name="text">
      <string>Run Probe Finding</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="runPositionCalibrationCheckBox">
     <property name="text">
      <string>Run Position Calibration</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="logPushButton">
     <property name="text">
      <string>Log: NONE</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="donePushButton">
     <property name="text">
      <string>Continue</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
